### PR TITLE
(HDS-2109) Add links to Storybooks to site.

### PR DESCRIPTION
**Pull request by mrTuomoK**
## Description

Add links to Storybooks to site.

## Related Issue

[HDS-2109](https://helsinkisolutionoffice.atlassian.net/browse/HDS-2109)

## How Has This Been Tested?

- local machine

## Add to changelog
- [x] Added needed line to changelog

[HDS-2109]: https://helsinkisolutionoffice.atlassian.net/browse/HDS-2109?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: site/src/docs/about/resources.mdx [hunk 1]
```diff
@@ -35,3 +35,7 @@ import InternalLink from '../../components/InternalLink';
 - <ExternalLink href="https://github.com/City-of-Helsinki/helsinki-design-system/tree/master/packages/react">HDS React</ExternalLink>
 - <ExternalLink href="https://github.com/City-of-Helsinki/helsinki-design-system/tree/master/packages/design-tokens">HDS Design tokens</ExternalLink>
 - <ExternalLink href="https://github.com/City-of-Helsinki/helsinki-design-system/tree/master/site">HDS Documentation site</ExternalLink>
+
+## HDS storybook examples
+- <InternalLink href="https://hds.hel.fi/storybook/react">React Storybook</InternalLink>
```
Comment (NikoHelle) [inline site/src/docs/about/resources.mdx:40]: As they are internal links, the `href` should be a relative path like in other internallinks:  `<InternalLink href="/storybook/react"`

**Review by NikoHelle — CHANGES_REQUESTED** [review]

File: site/src/docs/about/resources.mdx (hunk 1 above)
Comment (mrTuomoK) [reply site/src/docs/about/resources.mdx:40]: Good point, I'll fix

**Comment by mrTuomoK** [comment]
Ping @NikoHelle all should be OK now?

**Review by Riippi — APPROVED** [review]

**Review by laurakarhu — APPROVED** [review]
Thx!

**Pull request merged**